backend/src/main/java/com/woowacourse/levellog/prequestion/application/PreQuestionService.java

```diff
@@ -0,0 +1,137 @@
+package com.woowacourse.levellog.prequestion.application;
+
+import com.woowacourse.levellog.common.exception.InvalidFieldException;
+import com.woowacourse.levellog.common.exception.UnauthorizedException;
+import com.woowacourse.levellog.levellog.domain.Levellog;
+import com.woowacourse.levellog.levellog.domain.LevellogRepository;
+import com.woowacourse.levellog.levellog.exception.LevellogNotFoundException;
+import com.woowacourse.levellog.member.domain.Member;
+import com.woowacourse.levellog.member.domain.MemberRepository;
+import com.woowacourse.levellog.member.exception.MemberNotFoundException;
+import com.woowacourse.levellog.prequestion.domain.PreQuestion;
+import com.woowacourse.levellog.prequestion.domain.PreQuestionRepository;
+import com.woowacourse.levellog.prequestion.dto.PreQuestionDto;
+import com.woowacourse.levellog.prequestion.exception.PreQuestionNotFoundException;
+import com.woowacourse.levellog.team.domain.Participant;
+import com.woowacourse.levellog.team.domain.ParticipantRepository;
+import com.woowacourse.levellog.team.domain.Team;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class PreQuestionService {
+
+    private final PreQuestionRepository preQuestionRepository;
+    private final LevellogRepository levellogRepository;
+    private final MemberRepository memberRepository;
+    private final ParticipantRepository participantRepository;
+
+    @Transactional
+    public Long save(final PreQuestionDto request, final Long levellogId, final Long memberId) {
+        final Levellog levellog = getLevellog(levellogId);
+        final Member from = getMember(memberId);
+
+        validateIsAuthor(levellog, from);
+        validateExistParticipantByMember(levellog.getTeam(), from);
```

Comment:
메서드 이름이 이해하기 좀 어려운 것 같아요.

```suggestion
        validateSameTeamMember(levellog.getTeam(), from);
```
요런건 어떨까요?? 이건 적용하지 않으셔도 됩니다 😊

Reply:
더 짧고 명확해진 것 같아서 적용했습니다